Add 1040 to noise table in pseudopeople docs (#235)
ihmeuw/pseudopeople: docs/source/noise/index.rst

    - Types of noise
    - Notes
  * - First name
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
    - Leave a field blank, use a fake name, nickname, make typos, make OCR errors, make phonetic errors

>     - In the 1040 form, this noise type functions the same for joint filer and dependent *n*

Reviewer: I think "noise type" isn't very clear in this context, since this table is about columns each of which have _multiple_ noise types. I'm also not sure our users will get "dependent n".

Here's one idea:

```suggestion
    - In the 1040 form, the same noise types are applied to the corresponding columns for the joint filer and dependents
```

>     - Leave a field blank, use a fake name, nickname, make typos, make OCR errors, make phonetic errors

Reviewer 2: ```suggestion
    - Leave a field blank, use a fake name, use a nickname, make typos, make OCR errors, make phonetic errors
```
Looks like another merge conflict that didn't get resolved correctly

>     - In the 1040 form, the same noise types are applied to the corresponding columns for the joint filer and dependents

Reviewer 2: ```suggestion
    - In the 1040 form, the same noise types apply to the first name columns for the joint filer and dependents
```
Which noise types are actually applied to the columns depends on the configuration specified by the user, so I changed the wording to try to convey that the same noise types are _available_ for these other columns. I'm open to other wording suggestions.

Also, I think it would be clearer to specify which column we mean, since this row in the table applies to a specific column name.

  * - Middle initial
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
    - Leave a field blank, make typos, make OCR errors, make phonetic errors

>     - In the 1040 form, this noise type functions the same for joint filer and dependent *n*

Reviewer: Note: you'll want to change these rows as well to be consistent

>     - In the 1040 form, this noise type functions the same for joint filer and dependent *n*

Reviewer 2: ```suggestion
    - In the 1040 form, the same noise types apply to the middle initial columns for the joint filer and dependents
```
As Zeb noted, there are several rows that need to be updated to be consistent. Here's an update that matches my suggestion for the first name column.

  * - Last name
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
    - Leave a field blank, use a fake name, make typos, make OCR errors, make phonetic errors
    - In the 1040 form, the same noise types apply to the last name columns for the joint filer and dependents
  * - Age

>     - Decennial Census, ACS, CPS

Reviewer 2: ```suggestion
    - Decennial Census, ACS, CPS, W-2 and 1099
```
Removing age column from W2/1099 [hasn't been implemented yet](https://jira.ihme.washington.edu/browse/MIC-4134), and I don't know whether the change will make it into yellow release, so let's not make this change in the docs yet.

    - Leave a field blank, misreport age, make typos, make OCR errors
    -
  * - Date of birth

>     - Decennial Census, ACS, CPS, WIC, SSA

Reviewer 2: ```suggestion
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
```
Same comment as above -- let's not make this change yet since removing DOB hasn't been implemented.

    - Decennial Census, ACS, CPS, WIC
    - Leave a field blank, choose the wrong option
    -

>   * - SSN or ITIN

Reviewer: Do we need to update this on other pages where we reference SSN?

Reviewer 2: I think this cell should still just say "SSN" because that's the name of the column -- none of the datasets have a column called "ITIN". It's just that in the 1040, some of the values in the SSN column will actually be ITINs instead of SSNs, but that's not particularly relevant to the noise functions.

    - W-2 and 1099, 1040, SSA
    - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors

>     - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset; ITIN only included if no SSN present.

Reviewer: ```suggestion
    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.
      ITIN is only reported in the 1040, and only when the simulant does not have an SSN.
```

Maybe? I want to make it clear that there won't be ITIN in any other data source.

    - Decennial Census, ACS, CPS, WIC
    - Leave a field blank, choose the wrong option
    -
  * - SSN or ITIN
    - W-2 and 1099, 1040, SSA
    - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors
    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.

>       ITIN is only reported in the 1040, and only when the simulant does not have an SSN.
>       In the 1040 form, this noise type functions the same for joint filer and dependent *n*

Reviewer 2: ```suggestion
      In the 1040 form, the same noise types apply to the SSN columns for the joint filer and dependents
```
As per my other comment, the column name should just be listed as "SSN" (not "SSN or ITIN"), so I don't think it's necessary to mention ITIN at all on the noise page. For the second sentence, here's a suggestion to make this row consistent with my other suggestions above.

    - Leave a field blank, choose the wrong option
    -
  * - SSN
    - W-2 and 1099, 1040, SSA
    - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors

>     - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.

Reviewer 2: This is missing the changes made in https://github.com/ihmeuw/pseudopeople/pull/237.

>     - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos, make OCR errors
>     - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset.

Reviewer 2: ```suggestion
    - Borrow a social security number, leave a field blank, write the wrong digits, make typos, make OCR errors
    - Note that 'borrow a social security number' only applies to the W-2 and 1099 dataset.
```
This is missing the changes in https://github.com/ihmeuw/pseudopeople/pull/237 -- looks like the merge conflict wasn't resolved properly.

Author: Ah my bad, thanks for catching!
